src/components/PostDetails.tsx

```diff
@@ -2,91 +2,41 @@ import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
+import * as commentsAction from '../features/commets';
```

Comment:
There is a typo in the import statement for commentsAction. It should be imported from '../features/comments' instead of '../features/commets'. This will cause an error when trying to use commentsAction.